internal/expr/prepare.go

```diff
@@ -99,16 +107,15 @@ func prepareOutput(ti typeNameToInfo, p *outputPart) ([]fullName, []typeMember,
 	// Check target struct type and its tags are valid.
 	var info typeInfo
 	var ok bool
-	var err error
 
 	fetchInfo := func(typeName string) (typeInfo, error) {
 		info, ok := ti[typeName]
 		if !ok {
 			ts := getKeys(ti)
 			if len(ts) == 0 {
-				return nil, fmt.Errorf(`type %q not passed as a parameter`, typeName)
+				return nil, fmt.Errorf(`type %q not passed as a parameter. Type mentioned`, typeName)
 			} else {
-				return nil, fmt.Errorf(`type %q not passed as a parameter, have: %s`, typeName, strings.Join(ts, ", "))
+				return nil, fmt.Errorf(`type %q not passed as a parameter, have: %s. Type mentioned`, typeName, strings.Join(ts, ", "))
```

Comment:
This looks incomplete, but I'm also not sure whether this will be removed due to other reviews

Reply:
Removed already.